install/0000_90_machine-config-operator_01_prometheus-rules.yaml

```diff
@@ -43,6 +45,7 @@ spec:
           expr: |
             mcc_drain_err > 0
```

Comment:
Just curious, you have 2 commits in the PR, the first sets this to `mcc_drain_err => 0`, and the second sets it back to `mcc_drain_err > 0`. Was that for testing purposes? If so could you squash the commits together?

Otherwise lgtm! Thanks for looking into this!

Reply:
Agree with Jerry's assessment. Other than that lgtm.

Reply:
Yep, it was for just testing purposes when I was trying to emit the error